File: Python/LengthOfLastWord.py
word = "   Hello    World   "

def lengthOfLastWord(word):             # This is another basic problem.
    word = word.strip()                 # strip() method is used to erradicate the trailing spaces.
    count = 0                           
    for i in word[::-1]:                # string is accessed in reverse order and the no.of letters are counted.
        if i==' ': break
        else: count+=1
    return count
print(lengthOfLastWord(word))

# Revised Today (7 NOV 2025)

File: Java/LengthOfLastWord.java
// Leetcode : 58

public class LengthOfLastWord {
    public int lengthOfLastWord(String s) {
        s = s.strip();
        int count = 0;
        for(int i=s.length()-1; i>=0; i--){
            if(s.charAt(i)!=' '){
                count++;
            }
            else{
                break;
            }
        }
        return count;
    }
}
